vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -873,16 +873,17 @@ def fused_topk(
 # This is used by the Deepseek-V2 and Deepseek-V3 model
 @torch.compile(dynamic=True, backend=current_platform.simple_compile_backend)
 def grouped_topk(
-    hidden_states: torch.Tensor,
-    gating_output: torch.Tensor,
-    topk: int,
-    renormalize: bool,
-    num_expert_group: int = 0,
-    topk_group: int = 0,
-    scoring_func: str = "softmax",
-    e_score_correction_bias: Optional[torch.Tensor] = None
+        hidden_states: torch.Tensor,
+        gating_output: torch.Tensor,
+        topk: int,
+        renormalize: bool,
+        num_expert_group: int = 0,
+        topk_group: int = 0,
+        scoring_func: str = "softmax",
+        e_score_correction_bias: Optional[torch.Tensor] = None,
+        share_fusion: int = 0,
+        routed_scaling_factor: float = 2.5
```

Comment:
This (`routed_scaling_factor`) is only used when `share_fusion != 0`, right?

If so, could you make it an `Optional[float]` and default it to `None` if so? I think there will be fewer footguns this way, especially when shared expert fusion is applied to other models